sdk/identity/identity/test/public/node/authorizationCodeCredential.spec.ts

```diff
@@ -117,12 +117,7 @@ describe("AuthorizationCodeCredential", function() {
           children: [
             {
               name: "Azure.Identity.AuthorizationCodeCredential-getToken",
-              children: [
-                {
-                  children: [],
-                  name: "/tenant/oauth2/v2.0/token"
-                }
-              ]
+              children: []
```

Comment:
Core-http changes do change the tracing contexts. I am not too worried about this for this hotfix.

Reply:
Is that because of core-http or because the core-tracing used ends up being updated?

Reply:
I’ve updated Identity to 1.0.0-preview.12 as part of this PR now! Let me know what you think 🌞